synapse/handlers/room_member.py

```diff
@@ -1008,21 +1008,73 @@ def __init__(self, hs):
         self.distributor.declare("user_joined_room")
         self.distributor.declare("user_left_room")
 
+    @defer.inlineCallbacks
+    def _check_room_complexity_remote(self, room_id, remote_room_hosts):
```

Comment:
it's a bit unclear whether "True" means "this room is too complex" or "I've checked the room, and it was fine".

Perhaps a rename to `_is_{remote,local}_room_too_complex` ? Or just spell it out in the docstring?

Reply:
renamed, makes more sense